Banners GA (#9656)
braze-inc/braze-docs: _docs/_help/help_articles/campaigns_and_canvas/know_before_send.md

- [**Best practices**]({{site.baseurl}}/user_guide/message_building_by_channel/whatsapp/whatsapp_best_practices/): Review our suggested WhatsApp best practices.

## Banners

#### Things to check
- **Banner dimensions:** Build your Banners using a fixed dimension element and test them in the editor.
- **Priority:** If you’re launching multiple Banners, you can manually set the priority for how each banner is displayed.

#### Things to know
**Liquid personalization:** Liquid personalization refreshes at every session start.
**Placement and Banner ratio:** Each Banner placement can be used in up to 10 campaigns in a workspace.  
**Clicks and impressions:** Clicks and impressions for Banners are tracked automatically with the SDK.

> **Connected Content calls:** Using Connected Content isn’t currently supported in Banners.  

Reviewer: As well as promo codes, catalog_items :rerender and promo codes are also not supported

Reviewer: Missing other things that aren't supported like Canvas integration, API-triggered and action-based campaigns, and Currents

> **Connected Content calls:** Using Connected Content isn’t currently supported in Banners.  

Reviewer 2: ```suggestion
**Limitations:**  Currently, the following features aren't supported: Canvas integration, API-triggered and action-based campaigns, Currents, Connected Content, promotion codes, user-controlled dismissals, and `catalog_items` using the [`:rerender` tag]({{site.baseurl}}/user_guide/personalization_and_dynamic_content/catalogs/using_catalogs/#using-liquid).
```

**Testing:** To display the test Banner, the device you’re using must be able to receive foreground push notifications.

> **Custom HTML:** Leverage JS bridge to log clicks when using custom HTML. Clicks are only logged automatically from the Banner button.

Reviewer: It'd be nice to have this JS Bridge link to the generic documentation requested as part of https://jira.braze.com/browse/BD-4562

> 

Reviewer: ```suggestion
**Requesting Placements:** Up to 10 placements can be returned to the SDK. Each placement will include the highest priority Banner a user is eligible for.
```

> **Custom HTML:** Leverage [JS bridge]({{site.baseurl}}/user_guide/message_building_by_channel/in-app_messages/traditional/customize/html_in-app_messages/#javascript-bridge) to log clicks when using custom HTML. Clicks are only logged automatically from the Banner button.

Reviewer 3: ```suggestion
**Custom HTML:** Leverage [JS bridge]({{site.baseurl}}/user_guide/message_building_by_channel/in-app_messages/traditional/customize/html_in-app_messages/#javascript-bridge) to log clicks when using custom HTML to define click actions, like links and buttons. Click actions are only logged automatically when using the pre-built components in the drag-and-drop editor.
```

> **Requesting Placements:** Up to 10 placements can be returned to the SDK. Each placement will include the highest priority Banner a user is eligible for.

Reviewer: ```suggestion
**Requesting Placements:** Up to 10 placements can be returned to the SDK per session. Each placement will include the highest priority Banner a user is eligible for.
```
